src/SetProtocol.ts

```diff
@@ -110,27 +110,27 @@ class SetProtocol {
    * Calculates inputs for `createSetAsync` for a given list of ERC20 token addreses, proportions of each,
```

Comment:
Add a blurb here about what the precision is for? We do this in `FactoryAPI#createSetAsync` for example